tests/integration/components/public/event-map-test.js

```diff
@@ -0,0 +1,34 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('public/event-map', 'Integration | Component | public/event map');
+
+test('it renders', function(assert) {
+  assert.expect(2);
+
+  this.render(hbs`{{g-map}}`);
```

Comment:
Again ... why are you rendering g-map and testing it ? We need to test the event-map component. Not g-map component. I'm sure ember would have generated a proper base test for you. I'm not sure why you decided to remove that.